packages/strapi-parts/src/ModalLayout/ModalLayout.js

```diff
@@ -23,22 +23,36 @@ const ModalContent = styled(Box)`
 `;
 
 export const ModalLayout = ({ onClose, labelledBy, ...props }) => {
+  const modalContentRef = useRef();
+
   //FIX ME (find a way to do it globally)
   useEffect(() => {
     const body = document.body;
     body.classList.add('lock-body-scroll');
+    document.addEventListener('mousedown', handleClickAway);
```

Comment:
why not to put a onCLick on the overlay directly instead of attaching the event to the whole document?

Reply:
Oh you are right !!